gradle.properties

```diff
@@ -184,7 +184,7 @@ ANDROIDX_TEST_SERVICE_VERSION=1.4.2
 ANDROIDX_UI_AUTOMATOR_VERSION=2.3.0
 ANDROIDX_WORK_MANAGER_VERSION=2.9.0
 CORE_LIBRARY_DESUGARING_VERSION=2.0.4
-FIREBASE_BOM_VERSION_MATCHER=32.8.1
+FIREBASE_BOM_VERSION_MATCHER=33.1.1
```

Comment:
Is it necessary to add this change to this PR? Thank you.

Reply:
Yes it is. Firebase internally uses .so libraries too with the same problem :D With this newer version they fixes it